Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -713,11 +721,13 @@ static void logProcessPoolState(const WebProcessPool& pool)
     };
     m_notifyTokens = WTF::compactMap(notificationMessages, [this](const ASCIILiteral& message) -> std::optional<int> {
         int notifyToken = 0;
-        auto status = notify_register_dispatch(message, &notifyToken, dispatch_get_main_queue(), ^(int token) {
+        auto status = notify_register_dispatch(message, &notifyToken, dispatch_get_main_queue(), [this, message](int token) {
```

Comment:
Do we need to Retain `this` to avoid it being collected when the notification runs?

Reply:
That is a good point! I updated the patch to use a weak pointer, instead.

Thanks for reviewing!